src/types/validate.go

```diff
@@ -277,6 +292,18 @@ func (chart ZarfChart) Validate() error {
 		err = errors.Join(err, fmt.Errorf(lang.PkgValidateErrChartVersion, chart.Name))
 	}
 
+	// Validate releaseName if provided, else fallback to chart.Name
+	// This also ensures that the release name is a valid Helm release name
+	// Lastly this same fallback mechanism happens in src/internal/packager/helm/chart.go:InstallOrUpgradeChart
+	releaseName := chart.ReleaseName
```

Comment:
Is it possible to move this logic into the `validateReleaseName` function?

Reply:
I assumed there wasn't any qualms with using named returns here, but let me know if you'd rather have the `err` var explicitly declared. https://github.com/zarf-dev/zarf/commit/2658ff43b0a040fca86e34dbe2e91388ffaa2a8c